projects/DSVT/dsvt/transforms_3d.py

```diff
@@ -0,0 +1,116 @@
+from typing import List
+
+import numpy as np
+from mmcv import BaseTransform
+
+from mmdet3d.registry import TRANSFORMS
+
+
+@TRANSFORMS.register_module()
+class DSVTObjectRangeFilter(BaseTransform):
```

Comment:
Can rename it as ObjectRangeFilter3D and also rename the other functions similarly